docs/src/main/sphinx/connector/jdbc-procedures.fragment

```diff
@@ -0,0 +1,6 @@
+Procedures
+^^^^^^^^^^
+
+* ``system.flush_metadata_cache()``
+
+  Flush JDBC Metadata caches.
```

Comment:
We should add an example usage since people will obviously ask how to use this when multiple catalogs are present.

Something like:

```rst
For example the below query will flush the metadata caches for the `jdbc_catalog_one`.

.. code-block:: sql

    USE jdbc_catalog_one.myschema;
    CALL system.flush_metadata_cache();
```

cc: @mosabua for his inputs.

Reply:
Applied